rundeckapp/grails-app/controllers/rundeck/interceptors/ApiVersionInterceptor.groovy

```diff
@@ -60,6 +64,16 @@ class ApiVersionInterceptor {
     }
 
     boolean before() {
+        if(params[SynchronizerTokensHolder.TOKEN_KEY]) {
+            tokenVerifierController.withForm {
+                tokenVerifierController.refreshTokens()
+                session.api_access_allowed = true
```

Comment:
i think we should change this so that the api_access_allowed is set only for the request, not the session.  otherwise this will persist in the session, right?